src/styles/Less/components/AdminLeaderboard.css

```diff
@@ -0,0 +1,156 @@
+.leader-board-section {
```

Comment:
Can we move this to LESS file so we're sticking with one way to Style? We can hop on zoom and run through it, Its a pretty simple fix.